packages/dds/tree/src/shared-tree/treeCheckout.ts

```diff
@@ -604,6 +664,19 @@ export class TreeCheckout implements ITreeCheckoutFork {
 		this.revertibles.delete(revertible);
 	}
 
+	private disposeRevertibleRoots(revision: RevisionTag): void {
```

Comment:
See the comment on the one usage of this. If that comment is correct, then this function does not need to exist.